subprojects/docs/src/docs/userguide/migration/upgrading_version_6.adoc

```diff
@@ -56,6 +56,12 @@ The migration can be performed per configuration and does not have to be done in
 The `buildId` field will not be populated by default to ensure that the produced metadata file remains unchanged when no build inputs are changed.
 Users can still opt in to have this unique identifier part of the produced metadata if they want to, see <<publishing_gradle_module_metadata.adoc#sub:gmm-reproducible,the documentation>>.
 
+[[jcenter_deprecation]]
+=== `jcenter()` convenience method is now deprecated
+
+JFrog recently [announced](https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/) the decommission of JCenter.
+Gradle emits a warning message when `jcenter()` is used in the repository configuration.
```

Comment:
I think this lacks the call to action - what do I do/how do I upgrade when I see this warning?
Maybe:
```
<...>
Gradle emits a deprecation warning when `jcenter()` is used in the repository configuration.
Consider using `mavenCentral()`, `google()` or a private `maven` repository instead. (and link to relevant sections in declaring_repositories)
```

Reply:
Thanks for the suggestion. I've adjusted the text.